src/main/java/pl/wedding/management/model/employee/EmployeeRowFactory.java

```diff
@@ -0,0 +1,17 @@
+package pl.wedding.management.model.employee;
+
+
+class EmployeeRowFactory {
+
+    static Employee converter(EmployeeDto employeeDto) {
```

Comment:
Dlaczego oznaczyłeś tą metodę jako statyczną? Wydaje mi się, że nie jest to konieczne. :)